test/e2e/data/kubetest/conformance-ipv6-ginkgo-v2.yaml

```diff
@@ -0,0 +1,12 @@
+ginkgo.focus: \[Feature\:Networking-IPv6\]
+ginkgo.skip: \[Feature\:SCTPConnectivity\]|\[Experimental\]
```

Comment:
Why are we skipping `[Experimental]` in ipv6 but not dual-stack? Do `[Experimental]`-tagged test scenarios require ipv4, and our k8s dual-stack clusters are configured in such a way that those ipv4-dependent tests can be reliably tested?

Reply:
Because there is no `[Experimental]` dual-stack test but there was one ipv6 one that was failing https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_cluster-api-provider-azure/4086/pull-cluster-api-provider-azure-conformance-dual-stack-with-ci-artifacts/1711771121797304320

I could add it to both but the dual-stack one would be a no-op, with ginkgo skip I prefer to only skip tests only as needed.